.github/workflows/pr-title-check.yml

```diff
@@ -2,13 +2,19 @@ name: PR Title Check
 
 on:
   pull_request_target:
-    types: [opened, edited, synchronize, reopened]
+    types:
+      - opened
+      - edited
+      - synchronize
```

Comment:
```suggestion
```

My understanding is that this is triggered when more commits are pushed.  We shouldn't need to rerun this notification just because there are more commits.